app/assets/stylesheets/components/_search_bar.scss

```diff
@@ -1,24 +1,21 @@
 .search-bar-container {
-  display: flex;
-  justify-content: center;
   width: 100%;
 }
 
 .search-bar-all {
   border: 1px solid $my-green;
   border-radius: 5px;
   box-shadow: none;
+  color: $black;
```

Comment:
Wouldn't we use $my-black?

Reply:
It's the algolia tropdown. Do we want it to use our colors as well?

Reply:
<img width="386" alt="Capture d’écran 2020-02-27 à 10 30 37" src="https://user-images.githubusercontent.com/1598346/75430850-39c7bf80-594c-11ea-8f5e-811312786a18.png">


Looks good.